### PR TITLE
Refactor get login and register form to promise

### PR DESCRIPTION
**Pull request by isalga**

### CODE REVIEW

File: src/services/ZeroKitAdapter.js [hunk 1]
```diff
@@ -26,29 +26,33 @@ class ZeroKitAdapter {
...
+				zkitLoginNode.removeChild(zkitLoginNode.firstChild);
+			}
+			const zKitLoginObject =
+				this.zeroKit
+					.then(zeroKit => zeroKit.getLoginIframe(zkitLoginNode));
+
+			const submit = function (zKitLogin, event) {
+				event.preventDefault();
+				this.login(zKitLogin, document.getElementById(loginFormIds.hcUserLogin).value)
+					.then(resolve)
```
Comment (Glumli) [inline src/services/ZeroKitAdapter.js]: Why don't you just return this.login()?

File: src/services/ZeroKitAdapter.js [hunk 2]
```diff
@@ -82,24 +85,28 @@ class ZeroKitAdapter {
...
+			while (zkitRegisterNode.firstChild) {
+				zkitRegisterNode.removeChild(zkitRegisterNode.firstChild);
+			}
+			const zKitRegistrationObject =
+				this.zeroKit.then(zeroKit => zeroKit.getRegistrationIframe(zkitRegisterNode));
+
+			const submit = function (zKitRegistration, cb, event) {
+				event.preventDefault();
+				const userAlias = document.getElementById(registrationFormIds.hcUserRegister).value;
+				this.register(zKitRegistration, userAlias)
```
Comment (Glumli) [inline src/services/ZeroKitAdapter.js]: Same as for login

File: src/services/ZeroKitAdapter.js (hunk 1 above)
Comment (isalga) [reply src/services/ZeroKitAdapter.js]: It does not resolve the promise that way

File: src/services/ZeroKitAdapter.js (hunk 2 above)
Comment (isalga) [reply src/services/ZeroKitAdapter.js]: same

**Pull request merged**